libipl/p10/common.C

```diff
@@ -366,7 +366,9 @@ void ipl_plat_clock_error_handler(
 
 	ffdc.hwp_errorinfo.hwcallouts.push_back(hwcallout_data);
 
-	ipl_error_callback(ipl_error_info{IPL_ERR_CLK, &ffdc});
+	// All error, which return an FFDC, should be type of IPL_ERR_PLAT,
+	// so that it can be parsed easily inside callback function.
```

Comment:
JS: I think we could add like below since libipl intention is, group all plat specific errors as "IPL_ERR_PLAT" and not for the libipl user flexibility.

```
Use "ipl_error_type" as "IPL_ERR_PLAT" since it is plat specific error and
the exact error code and description are included in the FFDC.RC and FFDC.RC_DESC members.
```
Please rephrase it, if you did not like it.

Reply:
done